components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -182,24 +196,49 @@ export default {
       if (this.terminalLinesQueue.length <= 0) {
         this.isShowProceed = false
         this.terminalMode = mode.hybridInput
-        this.$emit('addTextLine', { content: 'End of Command', class: '' })
-        this.$emit('addTextLine', {
-          content: '-----------------------------',
-          class: ''
-        })
       }
       this.scrollToBottom()
     },
     executeCommandText(lines) {
       // takes a list of processed text lines and puts the terminal in clickContinue mode to read through them
-      this.terminalLinesQueue = lines.reverse() // overwrite any other terminal lines that may exist
+      lines.push({ content: 'End of Command\n--------------------------' })
+      this.terminalLinesQueue = lines.reverse()
       this.terminalMode = mode.clickContinue
       this.isShowProceed = true
-      this.continueText() // add this in a nextTick if issues arise?
+      this.continueText()
+    },
+    // #endregion
+    // #region Files handling Functions
+    navigate(arg) {
```

Comment:
maybe `mavigateTo`?

Reply:
also you may consider changing `arg` to `target` or `targetFolder` or something like that